opm/simulators/flow/BlackoilModelParametersEbos.hpp

```diff
@@ -259,32 +259,32 @@ struct MaxNewtonIterationsWithInnerWellIterations<TypeTag, TTag::FlowModelParame
 };
 template<class TypeTag>
 struct MaxInnerIterMsWells<TypeTag, TTag::FlowModelParameters> {
-    static constexpr int value = 100;
+    static constexpr int value = 20;
 };
 template<class TypeTag>
 struct MaxInnerIterWells<TypeTag, TTag::FlowModelParameters> {
-    static constexpr int value = 50;
+    static constexpr int value = 20;
 };
 template<class TypeTag>
 struct ShutUnsolvableWells<TypeTag, TTag::FlowModelParameters> {
-    static constexpr bool value = false;
+    static constexpr bool value = true;
 };
 template<class TypeTag>
 struct AlternativeWellRateInit<TypeTag, TTag::FlowModelParameters> {
     static constexpr bool value = true;
 };
 template<class TypeTag>
 struct StrictOuterIterWells<TypeTag, TTag::FlowModelParameters> {
-    static constexpr int value = 99;
+    static constexpr int value = 6;
 };
 template<class TypeTag>
 struct StrictInnerIterWells<TypeTag, TTag::FlowModelParameters> {
-    static constexpr int value = 40;
+    static constexpr int value = 10;
 };
 template<class TypeTag>
 struct RegularizationFactorMsw<TypeTag, TTag::FlowModelParameters> {
     using type = GetPropType<TypeTag, Scalar>;
-    static constexpr type value = 1;
+    static constexpr type value = 100;
```

Comment:
This changes the equation we solve, so I think this modification should be studied separately.

Reply:
I agree, lets do this in two steps. I will re-do some of testing and report back.